doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -97,7 +98,10 @@ See X-Road terms and abbreviations documentation \[[TA-TERMS](#Ref_TERMS)\].
 
 ### 2.1 Supported Platforms
 
-The security server runs on the *Ubuntu Server 14.04 Long-Term Support (LTS)* operating system on a 64-bit platform. The security server software is distributed as .deb packages through the official X-Road repository at http://x-road.eu/packages/
+The security server runs on the following platforms:
+
+* Ubuntu Server 14.04 Long-Term Support (LTS) operating system on a 64-bit platform. The security server software is distributed as .deb packages through the official X-Road repository at https://artifactory.niis.org/xroad-release-deb/
+* Red Hat Enterprise Linux 7 (RHEL7) operating system on a 64-bit platform. The security server software is distributed as .rpm packages through the official X-Road repository at https://artifactory.niis.org/xroad-release-rpm/
```

Comment:
Although it is true that Red Hat security server is supported, I would leave out the mention here and implement the complete RHEL documentation (including installation instructions) in a separate task.